data/success_stories/robur.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Secure Internet Services with OCaml and MirageOS
+logo: success-stories/robur.svg
+card_logo: success-stories/white/robur.svg
+background: /success-stories/robur-bg.jpg
+theme: blue
+synopsis: "A worker-owned collective leverages OCaml and MirageOS to build secure, high-performance, and resource-efficient software solutions"
+url: https://robur.coop/
+priority: 7
+why_ocaml_reasons:
+- Static Typing with Strong Guarantees
+- High Performance
+- Modular Architecture
+- Predictability
+- Stable Ecosystem
+- Optimized Resource Usage
+---
+
+## Challenge
+Hannes’ journey began with a passion for improving system infrastructure while fostering a collaborative, non-hierarchical work environment. Initially focused on formal verification during his PhD studies, he found the process to be tedious and impractical for large-scale impact. In his academic work, he encountered dynamically typed languages often used for network stacks, which introduced significant inefficiencies. These languages required runtime checks to ensure safety, resulting in performance costs and errors that only surfaced during execution. Seeking an alternative, Hannes envisioned a better approach: building secure, high-performance systems within a collective that valued open collaboration and sustainable practices.
+
+## Result
+In 2018, Hannes and a group of like-minded peers established Robur, a worker-owned collective committed to advancing MirageOS and creating impactful software solutions. The team quickly demonstrated their ability to deliver results. For instance, they developed a high-performance OpenVPN implementation, securing funding in 2019 and again in 2023 to optimize performance and finalize the protocol. Another achievement involved maintaining and extending the QubesOS Firewall, originally developed in 2015, to strengthen the security of isolated virtual machine environments.
+
+Robur also contributed significantly to cryptographic tools and protocols, including Mirage-Crypto and a robust TLS stack. Beyond development, the team uses MirageOS unikernels to host essential services such as their own DNS and CalDav servers. This active use ensures the reliability of their work while directly contributing to the stability and maintenance of MirageOS. These successes reflect the power of their collective model, where decisions about funding, projects, and partnerships are made collaboratively, fostering a culture of shared responsibility.
+
```

Comment:
Maybe worth to link to our recent blog article about our finances https://blog.robur.coop/articles/finances.html